src/test/java/edu/uci/ics/cs221/analysis/punctuation/Team2PunctuationTokenizerTest.java

```diff
@@ -0,0 +1,50 @@
+package edu.uci.ics.cs221.analysis.punctuation;
+
+import edu.uci.ics.cs221.analysis.PunctuationTokenizer;
+import org.junit.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class Team2PunctuationTokenizerTest {
+
+    /**
+     *  test1 tests whether multiple newlines, tabs,together with spaces works
+     */
+    @Test
+    public void test1() {
+        String text = "UCI: \n \n a public research university located in Irvine, \t \t California!";
+        List<String> expected = Arrays.asList("uci", "public", "research", "university", "located","irvine","california");
```

Comment:
Colon (`':'`) is not specified as a delimiter punctuation. See: https://github.com/UCI-Chenli-teaching/spring19-cs221-project/blob/d3587226af776caf2751974264615e6057fb165b/src/main/java/edu/uci/ics/cs221/analysis/PunctuationTokenizer.java#L24
```suggestion
        List<String> expected = Arrays.asList("uci:", "public", "research", "university", "located","irvine","california");
```

Reply:
Hi, I have updated the mistakes, thank you!